pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -4069,7 +4069,7 @@ async def get_core(self, p1: int, p2: int):
     if deck_size == STARLET_SIZE_X:
       xs = "07975" # 1360-797.5 = 562.5
     elif deck_size == STAR_SIZE_X:
-      xs = "13375" # 1900-1337.5 = 562.5
+      xs = "13385" # 1900-1337.5 = 562.5, plus a manual adjustment of + 10
```

Comment:
should I tweak the starlet by the same amount?

more generally should there be a constants.py file containing a "core_hotel" dict? This would ensure that the get and put commands use the same value.

Reply:
could this be a constant at the top of the file? / what is the rationale for putting this in a separate file if it's only used in STAR.py?

Reply:
it's just a massive file 🙃  either way works for me.

Reply:
it's massive, but i don't really see a clean way of splitting it up. moving tiny things to different files seem more confusing to me